GoogleForCreators/web-stories-wp
# Code Quality: Remove `improvedAutosaves` feature flag code

## Context

Now that this has been enabled by default in #12272  in the last release we can remove the relevant code bits.



## Summary

Removed `improved_autosaves` feature flag



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Ensure Save and AutoSave functionality still works as expected 


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?

No



### Does this PR have a legal-related impact?

No



## Checklist




---



Fixes #12307
